services/repository/src/main/java/org/sagebionetworks/repo/web/service/EntityServiceImpl.java

```diff
@@ -797,15 +797,17 @@ public ActionRequiredList getActionsRequiredForDownload(Long userId, String enti
 
 	@Override
 	public ValidateDefiningSqlResponse validateDefiningSql(ValidateDefiningSqlRequest request) {
-		String definingSql = request.getDefiningSql();
-		EntityType entityType = EntityType.valueOf(request.getEntityType().name());
+		ValidateArgument.requiredNotBlank(request.getDefiningSql(), "definingSql");
```

Comment:
validate the request is not null.
since the request.getEntityType() is used need to validate it is not null.